plugins/out_cloudwatch_logs/cloudwatch_api.c

```diff
@@ -360,7 +368,7 @@ static int init_put_payload(struct flb_cloudwatch *ctx, struct cw_flush *buf,
     }
     // If we are missing the service name, the entity will get rejected by the frontend anyway
```

Comment:
Why are we filtering on account_id here? Isn't it an optional field?

Reply:
Good question, it's an optional field but the backend logic treats any entity without account ID as if they were sent by the sender account id so it's not effective for cross-account checking. 

For example, if user A vends logs to user B, but account id is not in entity, backend will think the entity is sent to customer A's account, so user B will incorrectly get the entity from user A. In this case we need to drop the entity on client-side to prevent account information leaking outside the current account.

Reply:
Got it.

Nit: Can we update this comment to mention that the account ID is also required to not get rejected by the frontend?